adapters/repos/db/lsmkv/bucket_threshold_test.go

```diff
@@ -40,7 +37,7 @@ func TestWriteAheadLogThreshold_Replace(t *testing.T) {
 	values := make([][]byte, amount)
 
 	walThreshold := uint64(4096)
-	tolerance := 0.5
+	tolerance := 4.
```

Comment:
Woah, didn't know you could leave off the trailing `0`. Cool

Reply:
Yeap, also found it recently. Since preceding 0 can be skipped (`.123`) it makes sense to skip trailing ones :)